### PR TITLE
feature: footer

### LINKED ISSUE #40: Footer
Refer the Figma design for the footer.

### PR DESCRIPTION
**Pull request by Kanishkk-1**
Fixes #40

### CODE REVIEW

File: app/footer.tsx [hunk 1]
```diff
@@ -1,3 +1,169 @@
...
+import Image from "next/image";
+import footerimg from "./footer_img/footerimg.png";
+import twitter from "./footer_img/twitter.svg.png";
+import fb from "./footer_img/facebook.svg.png";
+import linkedin from "./footer_img/linkedin.svg.png";
+import logo from "./footer_img/nitkkrlogo.png";
+import call from "./footer_img/call.png";
+import mail from "./footer_img/mail.png";
+import distance from "./footer_img/distance.png";
+import lang from "./footer_img/language.png";
```
Comment (SaketRaman2002) [inline app/footer.tsx:11]: please use svg instead of png

**Comment by SaketRaman2002** [comment]
Please move all the images in a folder called assets and put the assets folder in components

File: app/footer.tsx (hunk 1 above)
Comment (Kanishkk-1) [reply app/footer.tsx]: Done

**Comment by Kanishkk-1** [comment]
Resolved all the stated queries

File: app/page.tsx [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import WorkInProgress from '@/components/work-in-progress';
 
+
 export default function Home() {
-  return <WorkInProgress />;
+  return <WorkInProgress />
+  ;
```
Comment (GetPsyched) [inline app/page.tsx:6]: Uh?

File: app/footer.tsx [hunk 3]
```diff
@@ -1,3 +1,170 @@
...
+                NIT KKR IT Infrastructure Usage Policy
+              </li>
+              <li className=" font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]">
+                NIT KKR @NDL
+              </li>
+              <li className=" font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]">
+                Telephone Directory
+              </li>
+            </ul>
+          </div>
```
Comment (GetPsyched) [inline app/footer.tsx:83]: Make this into an object that is read from instead of manually initialising each list element

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: app/footer.tsx [hunk 4]
```diff
@@ -1,3 +1,222 @@
...
+  },
+];
+
+
+
+const aboutUsItems = [
+  { text: "About Us", style: "font-bold md:text-[24px] py-3 sm:text-[16px]" },
+  {
+    text: "Council of NITs",
+    style: "font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]",
```
Comment (GetPsyched) [inline app/footer.tsx:59]: It will be `text` and `link`. Style is common to all and won't be included in the object

**Comment by GetPsyched** [comment]
Typo earlier, move the assets to `public/assets/`. Also, rebase on master and format all these files.
```sh
git checkout master
git pull
git checkout feature/footer
git rebase master
```

**Review by GetPsyched — CHANGES_REQUESTED** [review]
Typo earlier, move the assets to `public/assets/`. Also, rebase on master and format all these files.
```sh
git checkout master
git pull
git checkout feature/footer
git rebase master
```

File: app/footer.tsx [hunk 5]
```diff
@@ -1,3 +1,180 @@
...
+          <ul className="flex items-center justify-center lg:flex-row flex-col">
+            <div className="flex flex-row">
+              <li className="px-3">© 2022 NIT KKR, Inc.</li>
+              <li className="px-3">Terms</li>
+            </div>
+            <li className="pl-3">Privacy (Updated 08/2022)</li>
+          </ul>
+        </div>
+
+        <div className=" flex   flex-row  items-center justify-center flex-shrink-0 py-5  lg:pr-10 sm:pr-4 ">
```
Comment (GetPsyched) [inline app/footer.tsx:164]: Please format and remove these issues
```suggestion
        <div className="flex flex-row items-center justify-center flex-shrink-0 py-5 lg:pr-10 sm:pr-4">
```

File: app/footer.tsx [hunk 6]
```diff
@@ -1,3 +1,180 @@
...
+const departmentsItems = [
+  { text: "Vigilance Corner", link: "" },
+  { text: "PRO", link: "" },
+  { text: "OBC & PWD Cell", link: "" },
+  { text: "SC/ST Cell", link: "" },
+  { text: "Join Alumni Association", link: "" },
+  { text: "User Login", link: "" },
+  { text: "Contact Us", link: "" },
+];
+function footer() {
```
Comment (GetPsyched) [inline app/footer.tsx:67]:
```suggestion

function footer() {
```

File: app/page.tsx (hunk 2 above)
Comment (GetPsyched) [reply app/page.tsx:6]: Fix this

File: styles/globals.css [hunk 7]
```diff
@@ -1,7 +1,12 @@
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:opsz,wght@9..40,200&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=DM+Serif+Display&display=swap');
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
 
+
+
+
```
Comment (GetPsyched) [inline styles/globals.css:9]: formatting?? fix please.

File: tailwind.config.ts [hunk 8]
```diff
@@ -13,6 +13,10 @@ const config: Config = {
         'gradient-conic':
           'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
       },
+           fontFamily: {
+        sans:['DM Sans', 'sans-serif'],
+        display : ['DM Serif Display', 'sans-serif'],
+    },
```
Comment (GetPsyched) [inline tailwind.config.ts:19]: rebase onto master, this has been added already.

File: app/footer.tsx [hunk 9]
```diff
@@ -1,3 +1,180 @@
-export default function Footer() {
-  return <></>;
+// import React from 'react'
```
Comment (GetPsyched) [inline app/footer.tsx:1]:
```suggestion
```

File: app/footer.tsx [hunk 10]
```diff
@@ -1,3 +1,180 @@
...
+import Image from "next/image";
+import footerimg from "../public/assets/footerimg.png";
```
Comment (GetPsyched) [inline app/footer.tsx:3]:
```suggestion

import footerimg from "../public/assets/footerimg.png";
```

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: styles/globals.css
Comment (GetPsyched) [inline styles/globals.css:1]: Don't commit any changes in this file.

File: .vscode/settings.json
Comment (GetPsyched) [inline .vscode/settings.json:1]: remove this.

File: app/footer.tsx [hunk 11]
```diff
@@ -1,3 +1,180 @@
...
+  { text: "Vigilance Corner", link: "" },
+  { text: "PRO", link: "" },
+  { text: "OBC & PWD Cell", link: "" },
+  { text: "SC/ST Cell", link: "" },
+  { text: "Join Alumni Association", link: "" },
+  { text: "User Login", link: "" },
+  { text: "Contact Us", link: "" },
+];
+
+function footer() {
```
Comment (GetPsyched) [inline app/footer.tsx:67]:
```suggestion
export default function footer() {
```

File: app/footer.tsx [hunk 12]
```diff
@@ -1,76 +1,21 @@
...
-
-const departmentsItems = [
-  { text: "Vigilance Corner", link: "" },
-  { text: "PRO", link: "" },
-  { text: "OBC & PWD Cell", link: "" },
-  { text: "SC/ST Cell", link: "" },
-  { text: "Join Alumni Association", link: "" },
-  { text: "User Login", link: "" },
-  { text: "Contact Us", link: "" },
-];
```
Comment (GetPsyched) [inline app/footer.tsx]: bring this back.

File: app/page.tsx [hunk 13]
```diff
@@ -1,5 +1,5 @@
-import WorkInProgress from '@/components/work-in-progress';
+import WorkInProgress from "@/components/work-in-progress";
 
 export default function Home() {
-  return <WorkInProgress />;
-}
+  return <WorkInProgress/>;
```
Comment (GetPsyched) [inline app/page.tsx:4]: what, stop editing this file

File: tailwind.config.ts (hunk 8 above)
Comment (GetPsyched) [reply tailwind.config.ts:19]: resolve this also

File: app/page.tsx (hunk 13 above)
Comment (GetPsyched) [reply app/page.tsx:4]: this is not resolved, why marked as resolved. if you're confused, ask me, stop fighting with Git.

File: app/page.tsx
Comment (GetPsyched) [inline app/page.tsx:1]: I said stop editing this file. Why is this still in the PR??

File: app/footer.tsx [hunk 14]
```diff
@@ -1,3 +1,178 @@
...
+              <CiGlobe />
+            </div>
+            <div className="pr-5">
+              <SlLocationPin />
+            </div>
+          </div>
+        </div>
+
+        <div className="sm:items-center flex lg:items-center justify-center origin-center rotate-90 lg:rotate-0">
+          <Image src={footerimg} className="h-[362px] w-[74px]" alt="" />
```
Comment (GetPsyched) [inline app/footer.tsx:101]: Add a good alt message, don't just put it there to remove the NextJS error

File: app/footer.tsx [hunk 15]
```diff
@@ -1,3 +1,178 @@
...
+          <div className="font-display text-[22px] pt-3">
+            National Institute of
+          </div>
+          <div className="font-display text-[22px]">
+            Technology, Kurukshetra
+          </div>
+          <div className="font-sans text-[16px] text-[#8B949E] pt-3">
+            WRXF+23G, NIT, Thanesar, Haryana
+          </div>
+          <div className="font-sans text-[16px] text-[#8B949E]">136119</div>
```
Comment (GetPsyched) [inline app/footer.tsx:82]: This is not how you divide a text into multiple lines. Put it in a single div and apply a max width

File: app/footer.tsx [hunk 16]
```diff
@@ -1,3 +1,178 @@
...
+            <div className="pr-5">
+              <MdMailOutline />
+            </div>
+            <div className="pr-5">
+              <CiGlobe />
+            </div>
+            <div className="pr-5">
+              <SlLocationPin />
+            </div>
+          </div>
```
Comment (GetPsyched) [inline app/footer.tsx:97]: Use `ul` and `li` tags for this, not divs.

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: app/footer.tsx [hunk 17]
```diff
@@ -1,3 +1,182 @@
-export default function Footer() {
-  return <></>;
+import Image from 'next/image';
+import footerimg from '../public/assets/footerimg.png';
```
Comment (GetPsyched) [inline app/footer.tsx:2]: Get these images from Figma instead of keeping it in the Git tree. See #45 for reference

File: app/page.tsx
Comment (GetPsyched) [reply app/page.tsx:1]: Is there any doubt? you've still not removed the diff of this file

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: app/page.tsx
Comment (Kanishkk-1) [reply app/page.tsx:1]: I'm sorry i'm not able to resolve this issue , could you please walk me through it ?

File: app/footer.tsx (hunk 17 above)
Comment (Kanishkk-1) [reply app/footer.tsx:2]: Okay so i should remove the public/assets folder ? And you want me to add the images through the urls from figma?

Comment (GetPsyched) [reply app/footer.tsx:2]: Yes.

File: app/page.tsx
Comment (GetPsyched) [inline app/page.tsx:1]: Run `npm run format` or however else you want to run prettier. And then commit the changes in this file. It should go back to the way it was and won't be shown in the diff.

**Comment by Kanishkk-1** [comment]
is it okay now ? i ran npm run format on that page and its now formatted.

**Comment by Kanishkk-1** [comment]
and yeah its not showing the the diff too

File: app/footer.tsx [hunk 18]
```diff
@@ -1,3 +1,189 @@
...
+  { text: 'Vigilance Corner', link: '' },
+  { text: 'PRO', link: '' },
+  { text: 'OBC & PWD Cell', link: '' },
+  { text: 'SC/ST Cell', link: '' },
+  { text: 'Join Alumni Association', link: '' },
+  { text: 'User Login', link: '' },
+  { text: 'Contact Us', link: '' },
+];
+
+export default function footer() {
```
Comment (GetPsyched) [inline app/footer.tsx:65]:
```suggestion
export default function Footer() {
```

File: app/footer.tsx [hunk 19]
```diff
@@ -1,3 +1,189 @@
...
+          </div>
+        </div>
+
+        <div className="flex items-center justify-center origin-center rotate-90 lg:rotate-0">
+          <Image
+            width={210}
+            height={350}
+            src="https://s3-alpha-sig.figma.com/img/aeae/a206/68c4e501ecf5c3911343c1f3d8428cdf?Expires=1707696000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=S22f7ZBENQn-7VFFtfnHencMUHU6jvAsYS9gcgivdkyDW2~TCXcFb9ByJoCACBymHLXh50v7hy7SEOi-yEJYm2Xvk~yqKrUMtfCkWRjfh9EDErVFYx7nMQ4kQOSFTqSQWeUs9qymatl-CMJV8nlc-GvQ~bG8Xlij24qfYl~L9mGzSqSWUtPUi6xcmRKcLYbVMdisnKbn5NsDrpZ8ASxHyfBzarMRHULKzDvQEgmep9BoCkaOY-5UwOJF9~Z-Puo~5lVBh3olPUJQud689q3h095Sd9Yv0lZ9HnzpesX0V689O5wCrPen5nSY2lOhL69pKpP4Ixk~ynkZtdRIZFskoQ__"
+            className="h-[360px]"
+            alt="A vertical image in footer"
```
Comment (GetPsyched) [inline app/footer.tsx:108]:
```suggestion
            alt="Design Glyph"
```

**Review by GetPsyched — CHANGES_REQUESTED** [review]
Fonts need to be the default but that's already done in another PR so we'll wait until that's merged. Please do the following small changes for now

**Comment by Kanishkk-1** [comment]
Done. Anymore changes?

**Comment by GetPsyched** [comment]
None from your end, thanks!

This PR will now wait for #45 and also a formal list of what needs to be included in the footer.

**Review by GetPsyched — CHANGES_REQUESTED** [review]
For a learning experience, try replacing every single div element with a suitable semantic HTML tag. Feel free to refer the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element) for this.

Also, the footer bg is breaking after the rebase, please resolve.

File: app/directors-corner.tsx [hunk 20]
```diff
@@ -1,25 +1,31 @@
 import Image from 'next/image';
 import Link from 'next/link';
-
```
Comment (GetPsyched) [inline app/directors-corner.tsx:3]: Don't remove this line

File: app/directors-corner.tsx [hunk 21]
```diff
@@ -1,25 +1,31 @@
...
-        <h2 className="my-auto">DIRECT</h2>
-        <Image
-          alt="O"
-          className="my-auto h-[44px] rotate-[90.66deg]"
-          height={44}
-          width={44}
-          src="https://s3-alpha-sig.figma.com/img/d003/e4ca/46983bb9c2b53ca4f74a6382f4bfd57d?Expires=1707696000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=PaN9iN-Wh5U0K4D3p08lFy2vBXAuRS7835uwTzaGDtTKWYL0PBZ0Gw4yVjKtcbQMHiscxyh03PgyA4EyTRS0~sXpO14m1RJT8ch27VGGr6fa9zBXWpttPuHgX52tI9S-sodxlmj20HnYYxNObqIDxEkvzgCFJiHsUjVwmxQfOWxwduL80aBRKVUa6bGGSm2V2A2Znl96nJ1yUDdxenBvrTNN4UCFiIJEwyvdr4jVDoWxoqfDveVUcYElzjBjTTYvphifTSeLjEn-uHbX8JmmW1HkqEpThkWsDUVkoGPk~vLBP8LY6fW3aoa7S1ionsMVjG09UsnDCfB7T4Up40LZ6g__"
-        />
-        <h2 className="my-auto">R&rsquo;S CORNER</h2>
+        <div className="hidden md:block">
```
Comment (GetPsyched) [inline app/directors-corner.tsx:9]: divs aren't allowed in the repo. I'd say edit the horses running component to allow taking in breakpoints

File: app/directors-corner.tsx [hunk 22]
```diff
@@ -1,25 +1,31 @@
...
+              alt="O"
+              className="my-auto h-[44px] rotate-[90.66deg]"
+              height={44}
+              width={44}
+              src="https://s3-alpha-sig.figma.com/img/d003/e4ca/46983bb9c2b53ca4f74a6382f4bfd57d?Expires=1707696000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=PaN9iN-Wh5U0K4D3p08lFy2vBXAuRS7835uwTzaGDtTKWYL0PBZ0Gw4yVjKtcbQMHiscxyh03PgyA4EyTRS0~sXpO14m1RJT8ch27VGGr6fa9zBXWpttPuHgX52tI9S-sodxlmj20HnYYxNObqIDxEkvzgCFJiHsUjVwmxQfOWxwduL80aBRKVUa6bGGSm2V2A2Znl96nJ1yUDdxenBvrTNN4UCFiIJEwyvdr4jVDoWxoqfDveVUcYElzjBjTTYvphifTSeLjEn-uHbX8JmmW1HkqEpThkWsDUVkoGPk~vLBP8LY6fW3aoa7S1ionsMVjG09UsnDCfB7T4Up40LZ6g__"
+            />
+            <h2 className="my-auto">R&rsquo;S </h2>
+          </div>
+          <h2>&nbsp;CORNER</h2>
+        </div>
```
Comment (GetPsyched) [inline app/directors-corner.tsx:25]: Breaking stuff into more divs is not how you make things responsive.

**Review by GetPsyched — COMMENTED** [review]
I'm gonna try my hand at this.

**Comment by Kanishkk-1** [comment]
i ran git pull -f  on my local branch feature/footer

**Comment by Kanishkk-1** [comment]
Are any more changes required ?

**Review by GetPsyched — CHANGES_REQUESTED** [review]
- Each column should stick to the top
- Responsiveness is not implemented properly. Use flex and justify

File: app/footer.tsx [hunk 23]
```diff
@@ -1,3 +1,200 @@
...
+  { text: 'Join Alumni Association', link: '' },
+  { text: 'User Login', link: '' },
+  { text: 'Contact Us', link: '' },
+];
+
 export default function Footer() {
-  return <></>;
+  return (
+    <>
+      <div className="flex flex-col justify-center bg-[#000] md:flex-row lg:items-center">
```
Comment (GetPsyched) [inline app/footer.tsx:65]: Will not merge any div tags. Please refer https://developer.mozilla.org/en-US/docs/Web/HTML/Element and update to the most relevant semantic tags.

File: app/footer.tsx [hunk 24]
```diff
@@ -1,3 +1,200 @@
+import Image from 'next/image';
+import { CiGlobe } from 'react-icons/ci';
+import { FaFacebookSquare, FaLinkedinIn, FaTwitter } from 'react-icons/fa';
+import { MdMailOutline, MdOutlineCall } from 'react-icons/md';
+import { SlLocationPin } from 'react-icons/sl';
+
+const quickLinks = [
```
Comment (GetPsyched) [inline app/footer.tsx:7]: These objects will be defined inside the Footer function, not globally.

Also, compress all objects into one, like:
```
const items = {
    quickLinks = [ { text: "", link: "" ],
    about = [ { text: "", link: "" ],
    departments = [ { text: "", link: "" ],
}

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: app/footer.tsx [hunk 25]
```diff
@@ -1,3 +1,176 @@
...
+      { text: 'Join Alumni Association', link: '' },
+      { text: 'User Login', link: '' },
+      { text: 'Contact Us', link: '' },
+    ],
+  };
+
+  return (
+    <>
+      <section className="flex flex-col justify-center bg-[#000] md:flex-row lg:items-center">
+        <blockquote className="items flex flex-col items-center justify-center px-10">
```
Comment (GetPsyched) [inline app/footer.tsx:42]: Huh? How is this a blockquote? Please read what a tag is supposed to represent:
> The <blockquote> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element indicates that the enclosed text is an extended quotation.

File: app/footer.tsx [hunk 26]
```diff
@@ -1,3 +1,176 @@
...
+            src="https://s3-alpha-sig.figma.com/img/18b7/a13d/8bbb852e070e69b3de2a5ac59d20f501?Expires=1707696000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=gtaYqKeU8izdKAOskR5FvCKz15ZM9I~Fai49G9uVnIM1GLBs1e0sYvodH7rpjGkPIJ2yOczgOM6i~jECdaO3ufZHCUY~~80goh86AFkw~6vyJmKTmFVNUlz5kHfqMFmuaPqjTuoJY7XeFeDdYHTjrvGGNe6ATas9-IYPC2gSKavFZ8L5~tNY1vn-~IDnjdeemmKPOsyWbKDmUJWTfI~s~wsWLnKiwVFYpPjYZtwj-u1-~Dy2SuSFOKALF-AnZk1oSBIJrwnSS~Ox6aAP64YsclxQDO6NyJGWBbZP3s5jgVh6tHrHSzR9irkt9nyZj0ndSptaXYTRx4QFx8MYRfeDdw__"
+            alt="NIT KURUKSHETRA LOGO"
+            className="pt-5"
+            width={66}
+            height={66}
+          />
+          <header className="font-display max-w-[250px] pt-3 text-center font-serif text-[22px] text-[#fff]">
+            National Institute of Technology, Kurukshetra
+          </header>
+          <aside className="max-w-[250px] pt-3 text-center font-sans text-[16px] text-[#8B949E]">
```
Comment (GetPsyched) [inline app/footer.tsx:53]: This is not what aside is for. Use `address`

File: app/footer.tsx [hunk 27]
```diff
@@ -1,3 +1,176 @@
...
+            WRXF+23G, NIT, Thanesar, Haryana 136119
+          </aside>
+
+          <object>
```
Comment (GetPsyched) [inline app/footer.tsx:57]: Okay just what. Did you even read the docs. Looks like you've just asked ChatGPT to translate this.

> The <object> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents an external resource

"external resource"

File: app/footer.tsx [hunk 28]
```diff
@@ -1,3 +1,176 @@
...
+              <li className="pr-5">
+                <a href="">
+                  <SlLocationPin style={{ color: 'white', fontSize: '22px' }} />
+                </a>
+              </li>
+            </ul>
+          </object>
+        </blockquote>
+
+        <figure className="flex origin-center rotate-90 items-center justify-center lg:rotate-0">
```
Comment (GetPsyched) [inline app/footer.tsx:83]: This isn't required. Just use the image directly

File: app/footer.tsx [hunk 29]
```diff
@@ -1,3 +1,176 @@
...
+            src="https://s3-alpha-sig.figma.com/img/aeae/a206/68c4e501ecf5c3911343c1f3d8428cdf?Expires=1707696000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=S22f7ZBENQn-7VFFtfnHencMUHU6jvAsYS9gcgivdkyDW2~TCXcFb9ByJoCACBymHLXh50v7hy7SEOi-yEJYm2Xvk~yqKrUMtfCkWRjfh9EDErVFYx7nMQ4kQOSFTqSQWeUs9qymatl-CMJV8nlc-GvQ~bG8Xlij24qfYl~L9mGzSqSWUtPUi6xcmRKcLYbVMdisnKbn5NsDrpZ8ASxHyfBzarMRHULKzDvQEgmep9BoCkaOY-5UwOJF9~Z-Puo~5lVBh3olPUJQud689q3h095Sd9Yv0lZ9HnzpesX0V689O5wCrPen5nSY2lOhL69pKpP4Ixk~ynkZtdRIZFskoQ__"
+            className="h-[300px]"
+            alt="Design Glyph"
+            layout="fixed"
+          />
+        </figure>
+
+        <footer className="grid grid-cols-2 py-5 pl-5 lg:grid-cols-3">
+          <menu className="px-5 pb-5 lg:px-10">
+            <ul>
```
Comment (GetPsyched) [inline app/footer.tsx:96]: If you bother to read the docs, `menu` is a semantic _replacement_ for `ul`. You've nested both here.

File: app/footer.tsx [hunk 30]
```diff
@@ -1,3 +1,176 @@
...
+                >
+                  <a href={item.link}>{item.text}</a>
+                </li>
+              ))}
+            </ul>
+          </menu>
+        </footer>
+      </section>
+
+      <span className="flex items-center justify-between bg-[#101010]">
```
Comment (GetPsyched) [inline app/footer.tsx:143]: `span` tags aren't allowed either. This is also an incorrect usage of it.

File: app/footer.tsx [hunk 31]
```diff
@@ -1,3 +1,176 @@
...
+          <ul className="flex flex-col items-center justify-center lg:flex-row">
+            <menu className="flex flex-row">
+              <li className="px-3">© 2022 NIT KKR, Inc.</li>
+              <li className="px-3">Terms</li>
+            </menu>
+            <li className="pl-3">Privacy (Updated 08/2022)</li>
+          </ul>
+        </ol>
+
+        <figure>
```
Comment (GetPsyched) [inline app/footer.tsx:154]: Where's the figure here?

**Review by GetPsyched — CHANGES_REQUESTED** [review]
Please don't use ChatGPT to make the tags semantic. I could've done it myself, but I asked you to do it so you learn.

**Comment by Kanishkk-1** [comment]
?

**Comment by GetPsyched** [comment]
Am busy with mid semester exams ATM. Will review in some time. Meanwhile, please rebase on master to remove the merge conflicts.

**Review by GetPsyched — CHANGES_REQUESTED** [review]
Needs rebase to remove conflicts.

**Review by GetPsyched — APPROVED** [review]
@Kanishkk-1 please see the [files changed](https://github.com/nitkkr-dev/nitkkr/pull/47/files) to see how you can better your code quality.

Pro tip: Design the entire HTML before doing any styling. It forces you to use a minimal amount of tags instead of adding more tags when fighting with styling.

**Pull request merged**